Editor/Scripts/Analytics/MeshsyncAnalytics.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+using UnityEditor;
+
+using UnityEngine;
+
+namespace Unity.MeshSync.Editor.Analytics {
+
+    internal static class MeshSyncAnalyticsFactory {
+
+        public static IMeshSyncAnalytics CreateAnalytics() {
+            return new MeshSyncAnalytics();
+        }
+    }
+
+    internal interface IMeshSyncAnalytics {
+
+        /// <summary>
+        /// Send an event that a user installed and configured a DCC tool for use with MeshSync
+        /// </summary>
+        /// <param name="dccTool">Short designation for the DCC which was installed and configured</param>
+        void UserInstalledPlugin(string dccTool);
+
+        void UserSyncedData(MeshSyncAnalyticsData data);
+    }
+
+    internal sealed class MeshSyncAnalytics : IMeshSyncAnalytics {
+        private const string VENDORKEY = "unity.meshsync";
+
+        private const string DCCCONFIGURE_INSTALLEDEVENTNAME = "dccConfigure";
+        private const int DCCCONFIGURE_VERSION = 4;
```

Comment:
I would recommend to use an enum to give a bit of description what a particular number means.
Take a look at `SceneCacheClipDataVersion` as an example

Reply:
This refers to specific versions numbers of the data pipeline schema that was published in the internal Unity data pipeline. They are incremented as integers for every change made in the schema, it took 4 iterations to get the DCCConfigure event data correct, 2 iterations to get the Sync event data correct etc.

Reply:
Have added some comments to give some better context